CONVEX_MIGRATION.md

```
@@ -0,0 +1,93 @@
+# Migration to Convex
+
+This project has been migrated from Deno/MongoDB to Convex.
+
+## Setup Instructions
+
+1. **Create a Convex account**
+
+   - Visit https://dashboard.convex.dev
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)